src/org/locationtech/jts/algorithm/Angle.js

```diff
@@ -88,6 +91,38 @@ export default class Angle {
     if (angDel > Math.PI) return angDel - Angle.PI_TIMES_2
     return angDel
   }
+
+  /**
+     * Computes the angle of the unoriented bisector 
+     * of the smallest angle between two vectors.
+     * The computed angle will be in the range (-Pi, Pi].
+     * 
+     * @param tip1 the tip of v1
+     * @param tail the tail of each vector
+     * @param tip2 the tip of v2
+     * @return the angle of the bisector between v1 and v2
+     */
+  static bisector(tip1, tail,
+   tip2)
+  {
+  const angDel = this.angleBetweenOriented(tip1, tail, tip2);
+  const angBi = this.angle(tail, tip1) + angDel / 2;
```

Comment:
would this ever return an angle < -Pi or > Pi?   is the call to normalize() neccessary?

Reply:
good catch, it seems logically it would never be over pi or below -pi for angBi. However I have seen many of these unnecessary checks in JTS, I will just leave it be as this is how they are implemented in JTS.